nextcord/state.py

```diff
@@ -1405,6 +1408,44 @@ def create_message(
     ) -> Message:
         return Message(state=self, channel=channel, data=data)
 
+    def create_scheduled_event(
```

Comment:
There's a discrepancy between files. Some refer to them as `scheduled_event`, then others are just `guild_event` ..?

Reply:
could you show me where its called guild event?

Reply:
edited guild methods